include/functions.h

```diff
@@ -28,8 +28,8 @@ void DmaMgr_Error(DmaRequest* req, const char* file, const char* errorName, cons
 const char* DmaMgr_GetFileNameImpl(uintptr_t vrom);
 const char* DmaMgr_GetFileName(uintptr_t vrom);
 void DmaMgr_ProcessMsg(DmaRequest* req);
-void DmaMgr_ThreadEntry(void* arg);
-s32 DmaMgr_SendRequestImpl(DmaRequest* req, void* ram, uintptr_t vrom, u32 size, u32 unk, OSMesgQueue* queue, OSMesg msg);
+void DmaMgr_ThreadEntry(UNUSED void* arg);
```

Comment:
They don't need to, I thought binutils-gdb used it in prototypes but I just checked again and they don't..
I think it may be helpful to external users in the few cases where the purpose of the arg is unknown, marking it as unused informs them that they don't need to spend time trying to decipher what they should supply as that arg. I'm not too attached to it though.

Reply:
Right, and at the opposite end of the spectrum there are arguments where seeing `UNUSED` could be confusing, for example `TitleCard_InitBossName`'s `play`, maybe you'd think "so I pass NULL or what" idk

:shrug:

We can also leave it as is and reconsider later on a case-by-case basis
Stuff in functions.h is gonna move anyway (eventually :tm: )

See z64curve.h, the `UNUSED` in it is probably not needed in any way

Reply:
After letting this stew for a bit, I think most prototypes don't need UNUSED and it's only useful for documentation purposes in the case of (2) (the "unclear what this is" ones, cf https://github.com/zeldaret/oot/pull/1321#pullrequestreview-1035882290 again), and it could even be harmful in the case of (3) ("could have a use")

See: `z64curve.h`, `audio_load.c`, `audio_synthesis.c`, `z_demo.c`, `z_en_a_keep.c`, maybe others I missed (and `functions.h` but since those will move again I'm less concerned)

fwiw this is the only point of contention left for me but I'm ready to overlook it if others want it to be that way